flo-workflow/src/test/java/com/spotify/flo/ForwardingTaskContextGeneric.java

```diff
@@ -0,0 +1,53 @@
+/*-
+ * -\-\-
+ * Flo Workflow Definition
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.flo;
+
+public class ForwardingTaskContextGeneric<T> extends TaskContextGeneric<T> {
```

Comment:
Do we remember why `TaskContextGeneric` was not an interface to begin with?

Reply:
I do not remember. I believe we should try to make it an interface, but that's not in scope for this PR.